src/librustc_mir/tcx/expr.rs

```diff
@@ -345,16 +364,25 @@ impl<'tcx> Mirror<'tcx> for &'tcx hir::Expr {
             Some(&ty::adjustment::AdjustDerefRef(ref adj)) => {
                 for i in 0..adj.autoderefs {
                     let i = i as u32;
-                    let adjusted_ty =
-                        expr.ty.adjust_for_autoderef(
-                            cx.tcx,
-                            self.id,
-                            self.span,
-                            i,
-                            |mc| cx.tcx.tables.borrow().method_map.get(&mc).map(|m| m.ty));
+                    let adjusted_ty = expr.ty.adjust_for_autoderef(cx.tcx,
+                                                                   self.id,
+                                                                   self.span,
+                                                                   i,
+                                                                   |mc| {
+                                                                       cx.tcx
+                                                                         .tables
+                                                                         .borrow()
+                                                                         .method_map
+                                                                         .get(&mc)
+                                                                         .map(|m| m.ty)
```

Comment:
again with the `=`; something about these squashed up, long chains of field references really looks silly to me